release-notes/v4-tucker/4.6.15.md

```diff
@@ -0,0 +1,20 @@
+---
+title: 4.6.15
+---
+
+# 4.6.15
+
+11/5/2025
+
+- Audit logging robustness:
+  - Always record an audit entry when conflict/CRDT resolutions are performed so all blobs retain a reference for eventual cleanup
+  - Avoid referencing an audit entry when audit logging is disabled; ensure a new version is created when resolving a cache entry that replaces an existing one
+- Blob cleanup improvements:
+  - Reduce cleanup rate to limit resource usage under load
+  - Add a final "orphan cleanup finished" log message
+- Sharded deployments: do not connect to nodes with a database connection if the database is marked as sharded and is not in the same shard
+- Stability: clear the application timeout timer to allow threads to exit gracefully when still running
+- HTTP/router: fix missing `urlPath` option on initial entry handler creation (#2944, #2946)
```

Comment:
Lets remove these internal PR references.

Reply:
Good catch, thank you.